core/shared_queue.js

```diff
@@ -189,7 +195,9 @@ SharedQueue Binary Layout
       push,
       reset,
       shift
-    }
+    },
+    opsMap: {},
```

Comment:
The exposure of `Deno.core.opsMap` to public revision might be a security issue? E.g. people could sabotage by swapping `mkdir` and `remove` op IDs (our current permission prompt only warns about `read/write/net` attempts but not identifiable op names)

Honestly I'm not sure if this is actually really related to PR though. I start to feel that exposure of the message passing channel (aka `Deno.core`) is a potentially dangerous operation, and access to it might require privileges. (hiding `Deno` and `core` should work under careful handling, as seen in `Worker` `noDenoNamespace` case (where `setTimeout` still privately uses `Deno.core` features))

Reply:
It should definitely be read-only for users, but we still need to ability to update this map (eg. for plugin registration).